### PR TITLE
Add disclaimer notice on the competition overview page

### PR DESCRIPTION
**Pull request by DanielEgdal**
Disclaimer on the top of the competition page to alert people of the fact that currently there are less competitions than usual given the worldwide situation.

Let me know if there is an issue in the code. The integration between html and ruby is not quite clear to me.

### CODE REVIEW

File: WcaOnRails/app/views/competitions/index.html.erb [hunk 1]
```diff
@@ -3,6 +3,10 @@
 <div class="container">
   <h2><%= yield(:title) %></h2>
 
+  <div style="background-color: #ff0000 ; padding: 5px; border: 1px solid red; color:white;">
+    <h4><%= t('.disclaimer') %></h4>
+  </div>
```
Comment (jonatanklosko) [inline WcaOnRails/app/views/competitions/index.html.erb:8]: I think we can re-use the red alert we use in other places, so:

```erb
<%= alert :danger, t('.disclaimer') %>
```

Let me know if that looks appropriate =)

**Review by jonatanklosko — COMMENTED** [review]
Thanks for looking into that!

File: WcaOnRails/app/views/competitions/index.html.erb (hunk 1 above)
Comment (DanielEgdal) [reply WcaOnRails/app/views/competitions/index.html.erb]: That would make sense I think. Just pushed that change!

**Review by jonatanklosko — APPROVED** [review]
Great, thanks!

**Pull request merged**